src/taskpane/excel.js

```diff
@@ -3,6 +3,12 @@
  * See LICENSE in the project root for license information.
  */
 
+// images references in the manifest
+import "../../assets/icon-16.png";
```

Comment:
Is this really the new webpack standard? Seems weird to have to add all these assets files one by one.
I also don't like that their location is a development file. If we now need to add all these imports, I guess they could be in a separate file.

Reply:
This is not new . . . this just wasn't done in this template when it was done in all the others.  With my changes to the webpack files to be more standard across the templates it was exposed that these references weren't there.  Since some of these files get removed upon deployment of the template they need to be in each host file.

The reason they are needed is because of the manifest file.  When webpack process a file to deploy it, looks at the references and deploys those resources too and it updates references in those files to the new location deployed to by webpack (note the .css references in the html file) and does this recursively . . . but only for files that have a plug-in to do that.  The manifest file is just being deployed as a file and isn't processed in this way (not sure if there is a plugin or not that could do this) and so the reference there are not deployed automatically.  By putting image references in the js files we make sure the images get deployed by webpack.  Note this means they need to be in the location already indicated in the xml . . . thus the change to the webpack config file to keep the assets/* location (notice that the .css file doesn't need this).

It's likely that there are other ways to do this.  Making a separate import file would be one way.  I think I've seen instructions for a custom.d.js to do something like this as well.  Or, maybe there is a ways to have the manifest xml processed in the same way the html and other files are (automatically getting the assets referenced and dynamically updating the file if necessary).  For now, I'm just following the patter established in the other templates.

Reply:
Sure, it is fine for now. Could you create a TBD task about fixing this for the future?

Reply:
https://office.visualstudio.com/OC/_workitems/edit/5450116